build/ci-build.yml

```diff
@@ -59,6 +59,11 @@ stages:
               keepToken: false
               tokenPrefix: '#{'
               tokenSuffix: '}#'
+          - task: UseDotNet@2
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.PreviousVersion))'
+            inputs:
+              packageType: 'sdk'
+              version: '$(DotNet.Sdk.PreviousVersion)'
```

Comment:
Why do we need to import the previous version of the sdk ?

Reply:
Because the other project templates are not yet updated, and still uses this .NET Core version.

Reply:
Ah, ok.  And this build-ci is used for all templates.

Reply:
Yep indeed 😉